backends-gazelle/pom.xml

```diff
@@ -13,38 +13,17 @@
     <packaging>jar</packaging>
     <name>Gluten Backends Gazelle</name>
 
-    <profiles>
-        <profile>
-            <id>spark-3.2</id>
-            <activation>
-                <activeByDefault>true</activeByDefault>
-            </activation>
-            <dependencies>
-                <dependency>
-                    <groupId>io.glutenproject</groupId>
-                    <artifactId>spark-sql-columnar-shims-spark32</artifactId>
-                    <version>${project.version}</version>
-                    <scope>compile</scope>
-                </dependency>
-            </dependencies>
-        </profile>
-        <profile>
-            <id>spark-3.3</id>
-            <dependencies>
-                <dependency>
-                    <groupId>io.glutenproject</groupId>
-                    <artifactId>spark-sql-columnar-shims-spark33</artifactId>
-                    <version>${project.version}</version>
-                    <scope>compile</scope>
-                </dependency>
-            </dependencies>
-        </profile>
-    </profiles>
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+        <resource.dir>${project.basedir}/src/main/resources</resource.dir>
```

Comment:
Why specify resource dir? The default dir is this dir

Reply:
It's for targeting libgazelle_cpp.so.

Currently backends-gazelle reuse this setting from backends-velox.

Reply:
Could you please help find some references about this?

Reply:
https://maven.apache.org/plugins/maven-resources-plugin/examples/resource-directory.html
I think we can remove all the resource.dir

Reply:
I think I can raise another patch to remove all the `resource.dir` entries later on. Is that OK to you?